src/app/login-signup/login/login.component.html

```diff
@@ -1 +1,22 @@
-<p>login works!</p>
+
+    <div id="Login" class="background">
+
+        <form [formGroup]="loginForm">
+            <h3>Login Here</h3>
+        
+            <label for="username">Email</label><br>
```

Comment:
Wrap label and input in div tag to make proper sections of each.
Follow this to other sections too

Reply:
Changes are done